fhttp/http.go

```diff
@@ -523,9 +530,10 @@ func (c *BasicClient) Fetch() (int, []byte, int) {
 		log.Debugf("Reusing socket %v", *conn)
 	}
 	c.socket = nil // because of error returns
+	conErr := conn.SetReadDeadline(time.Now().Add(c.reqTimeout))
 	// Send the request:
 	n, err := conn.Write(c.req)
-	if err != nil {
+	if err != nil || conErr != nil {
```

Comment:
how did you test this ?
also this is too short for proxy requests in fetch, need to either raise it or set it higher in the uihandler
(see the circle ci failure and/or try:)

http://localhost:8080/fortio/fetch/localhost:8080/fortio/?url=http%3A%2F%2Flocalhost%3A8080%2Fecho&t=10s&load=Start

ie pasting `localhost:8080/fortio/?url=http%3A%2F%2Flocalhost%3A8080%2Fecho&t=10s&load=Start`
in the fetch box of http://localhost:8080/fortio/

will run for 10s thus timeout

Reply:
I had tested with TestHTTPRunner method. I put sleep before reading tcp socket to see is timeout working. Also, now I tested with url you gave. It is okey. When I set HTTPReqTimeOutDefaultValue to 10, it failed and when I set to 11 it worked.